script.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-implied-eval */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
```

Comment:
As both previous reviewers mentioned, you need to remove these lines as it is part of the requirements. Linters are helpful and show you some guidelines through constraints. The idea is that you as a programmer must search for different approaches to a problem. It's key to be resourceful and have initiative so that our programming skills get better rapidly. 

As you mentioned, you are having linter issues because you are calling the functions from HTML. There are many different approaches to solving a single problem but as an example, you could try using [BOM](https://www.w3schools.com/js/js_window.asp). By doing this, you assign your function to a global scope (window in the browser). 

![Captura2](https://user-images.githubusercontent.com/15807343/198096355-9370d454-4a90-43b6-a0d4-2c7d26728ba5.PNG)


![Captura1](https://user-images.githubusercontent.com/15807343/198096270-b80ae9de-c35a-40e2-be77-4804e45d09ee.PNG)

As you can see in the provided images, by using window.yourFunction = yourFunction, you get rid of the linters 🤓

Kindly make the necessary changes so that your project meets the requirements 📝